drivers/types.go

```diff
@@ -0,0 +1,41 @@
+package drivers
+
+var drivers map[string]WebhookDriver
+
+type DriverSchema struct {
+	Name        string            `json:"name"`
+	InputFormat map[string]string `json:"inputFormat"`
+}
+
+//RegisterDrivers creates object of type driver for every request
+func RegisterDrivers() {
+	drivers = map[string]WebhookDriver{}
+	drivers["serviceScale"] = &ServiceScaler{}
+}
+
+//GetDriver looks up the driver
+func GetDriver(key string) WebhookDriver {
+	if driver, ok := drivers[key]; ok {
```

Comment:
You dont need all the ok logic here. Just `return drivers[key]` will safely return nil if the key does not exist